app/main.py

```diff
@@ -1,13 +1,29 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {"errors": [{"line": error["line_number"],
+                        "column": error["column_number"],
+                        "message": error["text"],
+                        "name": error["code"],
+                        "source": "flake8"} for error in errors],
+            "path": file_path,
+            "status": "failed" if any(errors) else "passed"}
```

Comment:
There is no need for using `any`, there is no need for it
```suggestion
            "status": "failed" if errors else "passed"}
```